src/lib/review.js

```diff
@@ -9,7 +13,12 @@ async function askToReview(context) {
     })
   );
 }
-
+/**
+ * Calculates all the required reviewers needed for the PR and return them as an Array
+ * @param {string} context The context from which the PR is coming from
+ * @param {yaml} reviewersInfo The parsed yaml defined in reviewers.yml
```

Comment:
`yaml` is not supported by JSDoc https://jsdoc.app/tags-type.html
I would use just `Object` or `Object.<string, Object>` the `yaml` part is already defined in the param description, which is clear enough

Reply:
the same for label.getRequiredLables